testing/ansible-tester/Dockerfile

```diff
@@ -1,29 +1,69 @@
-FROM williamyeh/ansible:centos7
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
 
 ### Update system packages
-RUN yum --assumeyes --quiet upgrade
+RUN apt-get -q -y update
+RUN apt-get -q -y upgrade
+
+RUN apt-get install -q -y \
+    dmidecode \
+    python3 \
+    python3-pip \
+    gpg-agent \
+    rpm  \
+    openssh-client \
+    wget
 
-### Install additional ansible support
-RUN yum --assumeyes --quiet install dmidecode python3 rpm-build wget
 # XXX - The latest version of pip (21.3.1) is breaking on CentOS 7 without
 #       following encoding setup.
-ENV LANG=en_US.UTF-8
+ ENV LANG=en_US.UTF-8
 # XXX - ^^^
```

Comment:
Please verify if this is required for Ubuntu 22.04.  If it, update the message to show the current version of pip and the correct OS.

Reply:
Verified that current pip version on testing node is pip 22.0.2, everything is built correctly after commenting out `"ENV LANG=en_US.UTF-8"` Thank you for highlighting this.